### PR TITLE
spelling/grammar fixes part 4. Finished.

### PR DESCRIPTION
**Pull request by h00die**
Large spelling/grammar fixes in module descriptions.  This is the FINAL one.

## Verification
- [x] Check i didn't screw things up worserer

This is grouped with #8888 #8960 and #8940

### CODE REVIEW

File: modules/post/multi/gather/dns_srv_lookup.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ def initialize(info={})
     super( update_info( info,
         'Name'          => 'Multi Gather DNS Service Record Lookup Scan',
         'Description'   => %q{
-          Enumerates know SRV Records for a given domaon using target host DNS query tool.
+          Enumerates know SRV Records for a given domain using target host DNS query tool.
```
Comment (pbarry-r7) [inline modules/post/multi/gather/dns_srv_lookup.rb]: *known

Outside the scope of this original effort, but line 23 has output of 'ro' that should be 'to', methinks...

File: modules/post/multi/manage/system_session.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@ def initialize(info={})
         'Name'          => 'Multi Manage System Remote TCP Shell Session',
         'Description'   => %q{
           This module will create a Reverse TCP Shell on the target system
-          using the system own scripting enviroments installed on the
+          using the system own scripting environments installed on the
```
Comment (pbarry-r7) [inline modules/post/multi/manage/system_session.rb]: *system's

Also, I realize it wasn't part of this scope of effort, line 58 has a 'shel' in the output...

**Review by pbarry-r7 — COMMENTED** [review]
Herculean effort here on all these docs, @h00die.  Thank you!!!  :+1: :clap: :100:

**Comment by h00die** [comment]
fixed and fixed.
This isn't a Herculean effort. What @wvu-r7 performs every day are Herculean efforts.  This is more like easy

**Comment by sempervictus** [comment]
Me resent that. Work on mancave take great effort.

Thank you sir, we look less like asshats by the day.

File: modules/post/multi/manage/system_session.rb (hunk 2 above)
Comment (pbarry-r7) [reply modules/post/multi/manage/system_session.rb]: welp, sorry, meant the following line (13) "using the **system's** own scripting..."  :P

**Comment by sempervictus** [comment]
I believe American English deems the apostrophe for concatenation of "is" as opposed to an ownership indication. Though that seems to have changed over the course of my life - my grade school teachers explained it as interchangeably usable when I was learning the language (coming from one with no such thing it was never my favorite semantic).

**Comment by pbarry-r7** [comment]
As a native English speaker, it is (it's!) annoying and only gets worse when you bring plurals into the mix...  :P

**Comment by sempervictus** [comment]
In Soviet Russia, S apostrophes you, or worse if nobody is watching. Pluralized ownership by subjects ending with S is no fun, and I'm still confused about the moose/platypus cases of that.

**Comment by h00die** [comment]
I actually avoided working on apostrophes for this reason, and to avoid making potential code issues (needing escapes etc). It was enough to just do basic spelling and grammar with out needing to go to the next level.

**Comment by pbarry-r7** [comment]
Thanks, @h00die!

**Pull request merged**